include/SW_Weather.h

```diff
@@ -153,11 +157,23 @@ typedef struct {
     scale_temp_min[MAX_MONTHS],
     scale_skyCover[MAX_MONTHS],
     scale_wind[MAX_MONTHS],
-    scale_rH[MAX_MONTHS];
+    scale_rH[MAX_MONTHS],
+    scale_actVapPress[MAX_MONTHS],
+    scale_shortWaveRad[MAX_MONTHS];
 	char name_prefix[MAX_FILENAMESIZE - 5]; // subtract 4-digit 'year' file type extension
 	RealD snowRunoff, surfaceRunoff, surfaceRunon, soil_inf, surfaceAvg;
 	RealD snow, snowmelt, snowloss, surfaceMax, surfaceMin;
 
+  Bool use_cloudCoverMonthly, use_windSpeedMonthly, use_relHumidityMonthly,

```

Comment:
I suggest that we rename `use_relHumidityMonthly` to `use_humidityMonthly` -- because we need to be consistent, i.e., all humidity aspects are derived (consistently) from monthly inputs or none; see also comment to line 1364 `w->has_hurs = (w->use_relHumidityMonthly) ? swFALSE : w->has_hurs;` etc.